### PR TITLE
Document update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Proper punctuations

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/influxdb.md [hunk 1]
```diff
@@ -208,16 +208,16 @@ types of template variables.
 
 ### Query variable
 
-If you add a template variable of the type `Query` you can write a InfluxDB exploration (meta data) query. These queries can
+If you add a template variable of the type `Query` you can write an InfluxDB exploration (meta data) query. These queries can
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/influxdb.md:211]:
```suggestion
If you add a template variable of the type `Query`, then you can write an InfluxDB exploration (metadata) query. These queries can
```

File: docs/sources/features/datasources/influxdb.md [hunk 2]
```diff
@@ -208,16 +208,16 @@ types of template variables.
...
 return things like measurement names, key names or key values.
 
-For example you can have a variable that contains all values for tag `hostname` if you specify a query like this in the templating variable *Query* setting.
+For example, you can have a variable that contains all values for tag `hostname` if you specify a query like this in the templating variable *Query* setting.
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/influxdb.md:214]:
```suggestion
For example, you can have a variable that contains all values for tag `hostname` if you specify a query like this in the template variable **Query* setting.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Great changes! I have a couple more suggestions.

**Comment by NiteshKoushik** [comment]
Hello @oddlittlebird , Changes made as per the suggestion.

**Review by oddlittlebird — APPROVED** [review]
Excellent. Thanks for contributing to the docs!

**Pull request merged**